python/cudf/cudf/tests/test_reshape.py

```diff
@@ -529,3 +529,88 @@ def test_pivot_duplicate_error():
         gdf.pivot(index="a", columns="b")
     with pytest.raises(ValueError):
         gdf.pivot(index="b", columns="a")
+
+
+@pytest.mark.parametrize("aggfunc", ["mean", "count"])
```

Comment:
For `aggfunc`, do we support passing a list of functions or a dict that maps column names to aggregation functions? If so, can we add tests for those too?

Reply:
Added tests with `aggfunc` as a dict.